src/Model/Shared/OptionTranslation.php

```diff
@@ -20,18 +21,20 @@ class OptionTranslation extends BaseTranslation
     private $value;
 
     /**
-     * @return int
+     * @deprecated
+     * @return int|null
      */
-    public function getTicketCustomFieldOptionId(): int
+    public function getTicketCustomFieldOptionId(): ?int
     {
         return $this->ticketCustomFieldOptionId;
     }
 
     /**
-     * @param int $ticketCustomFieldOptionId
+     * @deprecated
+     * @param int|null $ticketCustomFieldOptionId
      * @return self
      */
-    public function setTicketCustomFieldOptionId(int $ticketCustomFieldOptionId): self
+    public function setTicketCustomFieldOptionId(?int $ticketCustomFieldOptionId): self
```

Comment:
Is there a bug reference / stack trace for this? I thought the issue was specifically to do with `null` `description` attributes on custom fields.

Reply:
There isn't a reference, the PR revises all translation models and fixes them. The initiator was the null/description.
The issue I detected here was having OptionTranslation linked to the TicketCustomfield where infact it is not. I only kept the methods here, and added nullability so code doesn't crash, and not to break BC.

These methods should be removed in the next version